### PR TITLE
[AST Builder] Implement Alter table statement 

### LINKED ISSUE #811: [AST Builder] Implement ALTER TABLE statement
## Description
implement `ast_builder` `alter table` statement

## Issue
related issue #614 

## Task
```rust
    #[test]
    fn alter_table() {
        let actual = table("Foo")
                .alter_table()
                .add_column("opt BOOLEAN NULL")
                .build();
        let expected = "ALTER TABLE Foo (opt BOOLEAN NULL)";
        test(actual, expected);

        let actual = table("Foo")
                .alter_table()
                .drop_column("col_name", false)
                .build();
        let expected = "ALTER TABLE Foo DROP COLUMN col_name";
        test(actual, expected);

        let actual = table("Foo")
                .alter_table()
                .rename_column("old", "new")
                .build();
        let expected = "ALTER TABLE Foo RENAME COLUMN old TO new";
        test(actual, expected);

        let actual = table("Foo")
                .alter_table()
                .rename_table("new_table_name")
                .build();
        let expected = "ALTER TABLE Foo RENAME TO new_table_name";
        test(actual, expected);
    }

```

### PR DESCRIPTION
**Pull request by ding-young**
## Issue
resolves #811 

## Description
`table node -> alter table node -> add col / drop col / rename col / rename table node . build()`

### CODE REVIEW

File: core/src/ast_builder/table.rs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 use super::{CreateTableNode, DeleteNode, DropTableNode, SelectNode, ShowColumnsNode, UpdateNode};
+use crate::ast_builder::alter_table::AlterTableNode;
```
Comment (ever0de) [inline core/src/ast_builder/table.rs:2]: Combine it with the import statement above
In addition, you need to attach the cfg of the alter-table feature.

File: core/src/ast_builder/mod.rs [hunk 2]
```diff
@@ -20,6 +22,9 @@ mod transaction;
 mod update;
 
 pub use {
+    alter_table::{
+        AddColumnNode, AlterTableNode, DropColumnNode, RenameColumnNode, RenameTableNode,
+    },
```
Comment (ever0de) [inline core/src/ast_builder/mod.rs:27]: ditto, I think you should put alter-table cfg.

**Review by ever0de — APPROVED** [review]
LGTM

**Review by panarch — APPROVED** [review]
All the codes look clean and nice! Thanks a lot 👍 👍

**Pull request merged**